java/arcs/android/sdk/host/AndroidHost.kt

```diff
@@ -49,18 +49,10 @@ abstract class AndroidHost(
 
     override val platformTime = JvmTime
 
-    @ExperimentalCoroutinesApi
-    override val stores: StoreManager = StoreManager(activationFactory)
-
     override fun onDestroy(owner: LifecycleOwner) {
         super.onDestroy(owner)
         runBlocking {
             shutdown()
         }
     }
-
-    override suspend fun shutdown() {
-        super.shutdown()
-        stores.reset()
```

Comment:
Is the idea here that the StoreManager provided by the StoreEndpointManager will be lifecycle aware? Keep in mind that super.shutdown() actually writes to storage, and so it needs to do work *before* any teardown down by StorageEndpointManager

Reply:
nvrmind, i see this is redundant because AbstractArcHost.shutdown() calls reset

Reply:
Yeah. It wasn't in `AbstractArcHost` originally because sometimes `stores` contained actual store objects, which we didn't want to close, but sometimes it actually contained storage *channels* which we did want to close. 

Now we provide `StorageEndpointManager` which is *always* providing channels. 

A subsequent work item will be to create an implementation of `StorageEndpointManager` that wraps some actual stores directly. That implementation will not close the stores when `reset` is called.